packages/admin/admin/src/actionLog/diffHeader/DiffHeader.tsx

```diff
@@ -0,0 +1,72 @@
+import { type Box, type ComponentsOverrides, type Theme, type Typography } from "@mui/material";
+import { useThemeProps } from "@mui/material/styles";
+import { FormattedDate, FormattedMessage } from "react-intl";
+
+import { type ThemedComponentBaseProps } from "../../helpers/ThemedComponentBaseProps";
+import { DateTypography, InfoTypography, Root, UserNameTypography, VersionTypography } from "./DiffHeader.sc";
+
+export type DiffHeaderClassKey = "root" | "versionTypography" | "userNameTypography" | "dateTypography" | "infoTypography";
+
+export interface DiffHeaderProps
+    extends ThemedComponentBaseProps<{
+        root: typeof Box;
+        versionTypography: typeof Typography;
+        userNameTypography: typeof Typography;
+        dateTypography: typeof Typography;
+        infoTypography: typeof Typography;
+    }> {
+    createdAt?: string;
+    userId?: string;
+    version?: number;
+}
+
+export const DiffHeader = (inProps: DiffHeaderProps) => {
+    const { createdAt, userId, version, slotProps, ...restProps } = useThemeProps({
+        props: inProps,
+        name: "CometAdminDiffHeader",
+    });
+
+    return (
+        <Root {...slotProps?.root} {...restProps}>
+            <VersionTypography {...slotProps?.versionTypography} color="white" variant="subtitle1">
```

Comment:
Generally, it should be possible to override the other props using `slotProps` (unless there is a specific reason it should never be possible to override a props value). 

```suggestion
            <VersionTypography color="white" variant="subtitle1" {...slotProps?.versionTypography}>
```

(Same goes for the other slots below)

Reply:
agree 👍 

✅ adjusted it.